CODEOWNERS

```diff
@@ -0,0 +1,14 @@
+# This file helps Github doing automatic review requests for new PRs.
+# It should always list the active maintainers of certain add-ons.
+
+# As a fallback, if no specific maintainer is listed below, assign the PR to the repo maintainers team:
+*       @openhab/2-x-add-ons-maintainers
```

Comment:
should be webui-maintainers here

Reply:
Didn't know this is a think. Where do I look that up for the future?

Reply:
You can click the arrow to expand the [teams](https://github.com/orgs/openhab/teams) and find the most likely team. Then if you hover over the URL you can see the the ID. 

If you're a repo Admin you can also hover over the team name in Settings > Collaborators & teams.

You can validate it by typing in a comment box `@openhab/<teamname>`

Reply:
> You can click the arrow

Ha! There's an arrow indeed 😄 Good catch!